components/molecule/tabs/src/index.scss

```diff
@@ -0,0 +1,168 @@
+@import '~@schibstedspain/sui-theme/lib/settings';
+@import '~@schibstedspain/sui-theme/lib/index';
+
+.sui-MoleculeTabs {
+  display: inline-flex;
+  &-scroller {
+    @include reset-list;
+    display: inline-flex;
+    flex-direction: row;
+    flex-wrap: nowrap;
+    justify-content: flex-start;
+    align-items: flex-start;
+  }
+  &--classic.sui-MoleculeTabs--horizontal, &--classic.sui-MoleculeTabs--fullWidth {
+    .sui-MoleculeTabs-button {
+      border-top: 1px solid transparent;
+      border-left: 1px solid transparent;
+      border-right: 1px solid transparent;
+      border-bottom: 1px solid $c-gray-light;
+      cursor: pointer;
+      &.is-active {
+        color: $c-primary;
+        border-top-color: $c-gray-light;
+        border-left-color: $c-gray-light;
+        border-right-color: $c-gray-light;
+        border-bottom-color:transparent;
+        svg {
+          fill: $c-primary;
+        }
+      }
+      &:hover:not(.is-disabled) {
+        color: $c-primary;   
+        svg {
+          fill: $c-primary;
+        }     
+      }
+    }
+  }
+  &--classic.sui-MoleculeTabs--vertical {
+    .sui-MoleculeTabs-button {
+      border-top: 1px solid transparent;
```

Comment:
Try to minimize statements
```css
border: 1px solid transparent;
border-right-color: $c-gray-light;
```

same for other classes

Reply:
Sure.